spring-cloud-data-module-deployers/spring-cloud-data-module-deployer-cloudfoundry/src/main/java/org/springframework/cloud/data/module/deployer/cloudfoundry/StandardCloudFoundryApplicationOperations.java

```diff
@@ -0,0 +1,233 @@
+/*
+ * Copyright 2015 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.data.module.deployer.cloudfoundry;
+
+import java.util.List;
+
+import org.springframework.web.client.RestClientException;
+
+/**
+ * Implementation of high-level operations on applications, limited to those
+ * operations required by the {@link CloudFoundryModuleDeployer}.
+ *
+ * @author Steve Powell
+ */
+class StandardCloudFoundryApplicationOperations implements CloudFoundryApplicationOperations {
```

Comment:
I think spring convention is to use `Operations` for an interface and then its implementation is `Template`. i.e. `RestOperations` vs. `RestTemplate`.